src/pages/team.js

```diff
@@ -1,25 +1,115 @@
-import React from "react"
-import { Card, Image } from 'semantic-ui-react'
+import React, { Component } from "react"
+import { Image, Container, Dropdown, Header } from 'semantic-ui-react'
 import { StaticQuery, graphql } from "gatsby"
+import Navbar from "../components/navbar"
+import Officer from "../components/officer"
 import data from '../data/team.json'
+import "../styles/team.css"
 
-const officers = data.officers;
-officers.map(officer => console.log(officer))
+class Team extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            activeTeam: 'Administrative'
+        }
+    }
 
+    handleTeamChange = (_, { value }) => {
+        this.setState({ activeTeam: value })
+    }
 
-const Team = () => (
-   <div>
-   <p>team page</p>
-   {officers.map(officer => 
-      <Card textAlign="center" key={officer.email}>
-         <Image src={officer.img} circular />
-         <Card.Content>
-         <Card.Header>{`${officer.name} '${officer.year}`}</Card.Header> 
-         <Card.Meta>{officer.team}</Card.Meta>
-         </Card.Content>
-      </Card>
-      )}
-   </div>
-)
+    render() {
+        const officers = data.officers.filter(officer => officer.team.includes(this.state.activeTeam))
+        const teamOptions = [
+            {
+                key: 'Administrative',
```

Comment:
why specify `key`, `text`, `value` if they are all going to be the same?

Reply:
i just tested but the key is not needed, `text` and `value` are needed for the semantic ui dropdown to work